### PR TITLE
versions doesn't have an empty? method.

### PR DESCRIPTION
**Pull request by DavidEBest**

### CODE REVIEW

**Comment by afeld** [comment]
Hmm, could we add a test that would fail without the change?

**Comment by DavidEBest** [comment]
@afeld Just added one. The tests _should_ have failed, but didn't because the stub object wasn't enough like the actual object.

File: spec/app_spec.rb [hunk 1]
```diff
@@ -49,11 +49,11 @@ def app
       name = 'good'
       StubComplianceData.any_instance.stubs(:base_name).returns(name)
       StubComplianceData.any_instance.stubs(:versions).returns(
-        [
+        StubClasses::StubCollection.new([
```
Comment (afeld) [inline spec/app_spec.rb:52]: Not a blocker, but we might want to look into using `instance_double(Aws::S3::Bucket)` here.

Comment (afeld) [inline spec/app_spec.rb:52]: Sorry, I mean `instance_double(Aws::Resources::Collection)`.

https://www.relishapp.com/rspec/rspec-mocks/v/3-4/docs/verifying-doubles/using-an-instance-double

**Pull request merged**